### PR TITLE
biolum color for glowshrooms is now inherited correctly

### LINKED ISSUE #11099: Allow glowshroom color to match its bioluminescence
Glowshrooms naturally have an off-green glow.
However if you plant them on floors or walls, they take 3 minutes to activate, then glow the regular light color. This is extremely lame, I say they should glow the color related to their biolum color.

### PR DESCRIPTION
**Pull request by Intigracy**
fixes https://github.com/d3athrow/vgstation13/issues/11099

### CODE REVIEW

**Comment by Probe1** [comment]
you of course tested it

**Comment by Ephx1** [comment]
nope

**Comment by Intigracy** [comment]
yep
thanks for reminding me prober it didn't fully work the way he had it

**Comment by kilozombie** [comment]
What will this do to the vox outpost on Box

**Comment by CptWad** [comment]
glowy

**Comment by Kurfursten** [comment]
@kilozombie Just looking at the code, it shouldn't affect the Vox outpost at all because those are roundstart clusters. This only changes the Spread() effect (which isn't called by anything afaik) and the planting of new clusters.

**Comment by Intigracy** [comment]
It makes planted glowshrooms not wait two minutes before they light up.

File: code/game/objects/effects/glowshroom.dm [hunk 1]
```diff
@@ -39,9 +39,10 @@
 	else //if on the floor, glowshroom on-floor sprite
 		icon_state = "glowshroomf"
 
-	spawn(delay)
-		set_light(round(potency/10))
-		// Spread() - Methinks this is broken - N3X
+	set_light(round(potency/10))
```
Comment (Intigracy) [inline code/game/objects/effects/glowshroom.dm:42]: Fug I shoulda spawn()'d this off now that I look at the affected code a bit more.

Comment (Probe1) [reply code/game/objects/effects/glowshroom.dm:42]: Rev up that PR merged count

**Pull request merged**